kiss_slam/kiss_slam_pybind/occupancy_mapper/occupancy_mapper.cpp

```diff
@@ -69,15 +72,18 @@ std::tuple<Vector3iVector, std::vector<float>> OccupancyMapper::GetOccupancyInfo
     return std::make_tuple(voxel_indices, voxel_occupancies);
 }
 
-Vector3iVector OccupancyMapper::GetOccupiedVoxels(float probability_threshold) const {
+Vector3iVector OccupancyMapper::GetOccupiedVoxels(
```

Comment:
you can remove the pass by reference `&` for the argument, just `const float` should be fine since float is only 4 bytes.